### DISCUSSION TITLE
Vertical federated learning

### DISCUSSION THREAD

**Discussion by mirsci** [NVIDIA/NVFlare#793] [Q&A] [answered]
Does NVidia Flare support federated learning for data partitioned **vertically**? 
If yes, could you please provide examples or tutorials for this scenario?

Thank you!

**Comment by holgerroth** [comment]
Yes, you can implement vertical FL workflow using the controller API. See existing examples [here](https://github.com/NVIDIA/NVFlare/tree/dev/nvflare/app_common/workflows).
We will provide examples/tutorials in the near future. 
cc @chesterxgchen.

**Reply by mirsci** [reply]
Thank you, @holgerroth. It will be great to see tutorials and more examples as well.

**Comment by mirsci** [comment]
Hello, 
are there any updates regarding vertical federated learning examples availability in Flare? 
Thank you!

**Comment by holgerroth** [comment, marked as answer]
Hi, thanks for reaching out. We are working on some split learning examples for the upcoming March release. There is a draft PR at the moment https://github.com/NVIDIA/NVFlare/pull/1168, but it is likely subject to change substantially until the release. So, better if you can wait a few weeks and check back once the PR is merged.

**Reply by mirsci** [reply]
Thank you so much, @holgerroth. Looking forward to see them in few weeks! 
It will be very helpful if these examples will cover structured/tabular data sets, as well.
